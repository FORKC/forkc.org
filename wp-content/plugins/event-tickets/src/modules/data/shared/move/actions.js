/* eslint-disable camelcase */
/**
 * Internal Dependencies
 */
import * as types from './types';

export const showModal = ( ticketId, blockId, ) => ( {
	type: types.SHOW_MODAL,
	payload: { ticketId, blockId },
} );

export const hideModal = () => ( {
	type: types.HIDE_MODAL,
} );

export const setModalData = ( payload ) => ( {
	type: types.SET_MODAL_DATA,
	payload,
} );
