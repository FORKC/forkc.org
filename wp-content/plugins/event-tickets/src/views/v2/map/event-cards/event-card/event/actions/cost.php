<?php
/**
 * View: Map View - Single Event Actions - Cost
 *
 * Override this template in your own theme by creating a file at:
 * [your-theme]/tribe/event-tickets/views/v2/map/event-cards/event-card/event/actions/cost.php
 *
 * See more documentation about our views templating system.
 *
 * @link {INSERT_ARTCILE_LINK_HERE}
 *
 * @version 4.10.9
 *
 * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
 *
 * @see tribe_get_event() For the format of the event object.
 *
 */
if ( empty( $event->tickets->exist ) ) {
	return;
}
?>

<a
	href="<?php echo esc_url( $event->tickets->link->anchor ); ?>"
	class="tribe-events-c-small-cta__link tribe-common-cta tribe-common-cta--thin-alt"
>
	<?php echo esc_html( $event->tickets->link->label ); ?>
</a>
