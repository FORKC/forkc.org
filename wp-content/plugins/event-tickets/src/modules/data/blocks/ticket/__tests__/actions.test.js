/**
 * Internal dependencies
 */
import * as actions from '../actions';

describe( 'Ticket actions', () => {
	const blockId = 'modern-tribe';

	describe( 'Tickets actions', () => {
		test( 'set tickets initial state', () => {
			const props = {
				clientId: 'modern-tribe',
				attributes: {
					header: 99,
					sharedCapacity: 19,
				},
			};
			expect( actions.setTicketsInitialState( props ) ).toMatchSnapshot();
		} );

		test( 'set tickets header image', () => {
			expect( actions.setTicketsHeaderImage( { image: 10 } ) ).toMatchSnapshot();
		} );

		test( 'set tickets is settings open', () => {
			expect( actions.setTicketsIsSettingsOpen( true ) ).toMatchSnapshot();
			expect( actions.setTicketsIsSettingsOpen( false ) ).toMatchSnapshot();
		} );

		test( 'set tickets is settings loading', () => {
			expect( actions.setTicketsIsSettingsLoading( true ) ).toMatchSnapshot();
			expect( actions.setTicketsIsSettingsLoading( false ) ).toMatchSnapshot();
		} );

		test( 'open settings', () => {
			expect( actions.openSettings() ).toMatchSnapshot();
		} );

		test( 'close settings', () => {
			expect( actions.closeSettings() ).toMatchSnapshot();
		} );

		test( 'set tickets provider', () => {
			expect( actions.setTicketsProvider( 'Tribe__Tickets__Commerce__PayPal__Main' ) ).toMatchSnapshot();
		} );

		test( 'set tickets shared capacity', () => {
			expect( actions.setTicketsSharedCapacity( 99 ) ).toMatchSnapshot();
		} );

		test( 'set tickets temp shared capacity', () => {
			expect( actions.setTicketsTempSharedCapacity( 99 ) ).toMatchSnapshot();
		} );
	} );

	describe( 'Header image saga actions', () => {
		test( 'fetch tickets header image', () => {
			expect( actions.fetchTicketsHeaderImage( 1 ) ).toMatchSnapshot();
		} );

		test( 'update tickets header image', () => {
			expect( actions.updateTicketsHeaderImage( { id: 1, alt: 'hi', src: '#' } ) ).toMatchSnapshot();
		} );

		test( 'delete tickets header image', () => {
			expect( actions.deleteTicketsHeaderImage() ).toMatchSnapshot();
		} );
	} );

	describe( 'Ticket details actions', () => {
		test( 'set ticket title', () => {
			expect( actions.setTicketTitle( blockId, 'Modern Tribe' ) );
		} );

		test( 'set ticket description', () => {
			expect( actions.setTicketDescription( blockId, 'The Next Generation of Digital Agency' ) )
				.toMatchSnapshot();
		} );

		test( 'set ticket price', () => {
			expect( actions.setTicketPrice( blockId, 99 ) ).toMatchSnapshot();
		} );

		test( 'set ticket sku', () => {
			expect( actions.setTicketSku( blockId, 'my-sku' ) ).toMatchSnapshot();
		} );

		test( 'set ticket start date', () => {
			expect( actions.setTicketStartDate( blockId, '2018-01-01' ) ).toMatchSnapshot();
		} );

		test( 'set ticket start date input', () => {
			expect( actions.setTicketStartDateInput( blockId, 'January 1, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set ticket start moment', () => {
			expect( actions.setTicketStartDateMoment( blockId, { type: 'moment' } ) ).toMatchSnapshot();
			expect( actions.setTicketStartDateMoment( blockId, null ) ).toMatchSnapshot();
		} );

		test( 'set ticket end date', () => {
			expect( actions.setTicketEndDate( blockId, '2018-01-10' ) ).toMatchSnapshot();
		} );

		test( 'set ticket end date input', () => {
			expect( actions.setTicketEndDateInput( blockId, 'January 10, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set ticket end moment', () => {
			expect( actions.setTicketEndDateMoment( blockId, { type: 'moment' } ) ).toMatchSnapshot();
			expect( actions.setTicketEndDateMoment( blockId, null ) ).toMatchSnapshot();
		} );

		test( 'set ticket start time', () => {
			expect( actions.setTicketStartTime( blockId, '10:00' ) ).toMatchSnapshot();
		} );

		test( 'set ticket end time', () => {
			expect( actions.setTicketEndTime( blockId, '12:34' ) ).toMatchSnapshot();
		} );

		test( 'set ticket start time input', () => {
			expect( actions.setTicketStartTimeInput( blockId, '10:00' ) ).toMatchSnapshot();
		} );

		test( 'set ticket end time input', () => {
			expect( actions.setTicketEndTimeInput( blockId, '12:34' ) ).toMatchSnapshot();
		} );

		test( 'set ticket capacity type', () => {
			expect( actions.setTicketCapacityType( blockId, 'unlimited' ) ).toMatchSnapshot();
		} );

		test( 'set ticket capacity', () => {
			expect( actions.setTicketCapacity( blockId, '10' ) ).toMatchSnapshot();
		} );
	} );

	describe( 'Ticket temp details actions', () => {
		test( 'set ticket temp title', () => {
			expect( actions.setTicketTempTitle( blockId, 'Modern Tribe' ) );
		} );

		test( 'set ticket temp description', () => {
			expect( actions.setTicketTempDescription( blockId, 'The Next Generation of Digital Agency' ) )
				.toMatchSnapshot();
		} );

		test( 'set ticket temp price', () => {
			expect( actions.setTicketTempPrice( blockId, 99 ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp sku', () => {
			expect( actions.setTicketTempSku( blockId, 'my-sku' ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp start date', () => {
			expect( actions.setTicketTempStartDate( blockId, '2018-01-01' ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp start date input', () => {
			expect( actions.setTicketTempStartDateInput( blockId, 'January 1, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp start moment', () => {
			expect( actions.setTicketTempStartDateMoment( blockId, { type: 'moment' } ) ).toMatchSnapshot();
			expect( actions.setTicketTempStartDateMoment( blockId, null ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp end date', () => {
			expect( actions.setTicketTempEndDate( blockId, '2018-01-10' ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp end date input', () => {
			expect( actions.setTicketTempEndDateInput( blockId, 'January 10, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp end moment', () => {
			expect( actions.setTicketTempEndDateMoment( blockId, { type: 'moment' } ) ).toMatchSnapshot();
			expect( actions.setTicketTempEndDateMoment( blockId, null ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp start time', () => {
			expect( actions.setTicketTempStartTime( blockId, '10:00' ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp end time', () => {
			expect( actions.setTicketTempEndTime( blockId, '12:34' ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp start time input', () => {
			expect( actions.setTicketTempStartTimeInput( blockId, '10:00' ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp end time input', () => {
			expect( actions.setTicketTempEndTimeInput( blockId, '12:34' ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp capacity type', () => {
			expect( actions.setTicketTempCapacityType( blockId, 'unlimited' ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp capacity', () => {
			expect( actions.setTicketTempCapacity( blockId, '10' ) ).toMatchSnapshot();
		} );
	} );

	describe( 'Ticket actions', () => {
		test( 'register ticket block', () => {
			expect( actions.registerTicketBlock( blockId ) ).toMatchSnapshot();
		} );

		test( 'remove ticket block', () => {
			expect( actions.removeTicketBlock( blockId ) ).toMatchSnapshot();
		} );

		test( 'set ticket sold', () => {
			expect( actions.setTicketSold( blockId, 23 ) ).toMatchSnapshot();
		} );

		test( 'set ticket availability', () => {
			expect( actions.setTicketAvailable( blockId, 99 ) ).toMatchSnapshot();
		} );

		test( 'set ticket id', () => {
			expect( actions.setTicketId( blockId, 99 ) ).toMatchSnapshot();
		} );

		test( 'set ticket currency symbol', () => {
			expect( actions.setTicketCurrencySymbol( blockId, '$' ) ).toMatchSnapshot();
		} );

		test( 'set ticket currency positioin', () => {
			expect( actions.setTicketCurrencyPosition( blockId, 'suffix' ) ).toMatchSnapshot();
		} );

		test( 'set ticket provider', () => {
			expect( actions.setTicketProvider( blockId, 'provider' ) ).toMatchSnapshot();
		} );

		test( 'set ticket is loading', () => {
			expect( actions.setTicketIsLoading( blockId, true ) ).toMatchSnapshot();
			expect( actions.setTicketIsLoading( blockId, false ) ).toMatchSnapshot();
		} );

		test( 'set ticket has been created', () => {
			expect( actions.setTicketHasBeenCreated( blockId, true ) ).toMatchSnapshot();
			expect( actions.setTicketHasBeenCreated( blockId, false ) ).toMatchSnapshot();
		} );

		test( 'set ticket has changes', () => {
			expect( actions.setTicketHasChanges( blockId, true ) ).toMatchSnapshot();
			expect( actions.setTicketHasChanges( blockId, false ) ).toMatchSnapshot();
		} );

		test( 'set ticket is selected', () => {
			expect( actions.setTicketIsSelected( blockId, true ) ).toMatchSnapshot();
			expect( actions.setTicketIsSelected( blockId, false ) ).toMatchSnapshot();
		} );
	} );

	describe( 'Ticket saga actions', () => {
		test( 'set ticket details', () => {
			expect( actions.setTicketDetails( blockId, {} ) ).toMatchSnapshot();
		} );

		test( 'set ticket temp details', () => {
			expect( actions.setTicketTempDetails( blockId, {} ) ).toMatchSnapshot();
		} );

		test( 'handle ticket start date', () => {
			expect( actions.handleTicketStartDate( blockId, {}, {} ) ).toMatchSnapshot();
		} );

		test( 'handle ticket start date', () => {
			expect( actions.handleTicketEndDate( blockId, {}, {} ) ).toMatchSnapshot();
		} );

		test( 'handle ticket start date', () => {
			expect( actions.handleTicketStartTime( blockId, 1000 ) ).toMatchSnapshot();
		} );

		test( 'handle ticket start date', () => {
			expect( actions.handleTicketEndTime( blockId, 1000 ) ).toMatchSnapshot();
		} );

		test( 'fetch ticket', () => {
			expect( actions.fetchTicket( blockId, 99 ) ).toMatchSnapshot();
		} );

		test( 'create new ticket', () => {
			expect( actions.createNewTicket( blockId ) ).toMatchSnapshot();
		} );

		test( 'update ticket', () => {
			expect( actions.updateTicket( blockId ) ).toMatchSnapshot();
		} );

		test( 'delete ticket', () => {
			expect( actions.deleteTicket( blockId ) ).toMatchSnapshot();
		} );

		test( 'set ticket initial state', () => {
			const props = {
				clientId: 'modern-tribe',
				attributes: {
					ticketId: 99,
				},
			};
			expect( actions.setTicketInitialState( props ) ).toMatchSnapshot();
		} );
	} );
} );
