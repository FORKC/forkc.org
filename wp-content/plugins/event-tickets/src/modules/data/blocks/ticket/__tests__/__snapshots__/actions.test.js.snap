// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ticket actions Header image saga actions delete tickets header image 1`] = `
Object {
  "type": "@@MT/TICKETS/DELETE_TICKETS_HEADER_IMAGE",
}
`;

exports[`Ticket actions Header image saga actions fetch tickets header image 1`] = `
Object {
  "payload": Object {
    "id": 1,
  },
  "type": "@@MT/TICKETS/FETCH_TICKETS_HEADER_IMAGE",
}
`;

exports[`Ticket actions Header image saga actions update tickets header image 1`] = `
Object {
  "payload": Object {
    "image": Object {
      "alt": "hi",
      "id": 1,
      "src": "#",
    },
  },
  "type": "@@MT/TICKETS/UPDATE_TICKETS_HEADER_IMAGE",
}
`;

exports[`Ticket actions Ticket actions register ticket block 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
  },
  "type": "@@MT/TICKETS/REGISTER_TICKET_BLOCK",
}
`;

exports[`Ticket actions Ticket actions remove ticket block 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
  },
  "type": "@@MT/TICKETS/REMOVE_TICKET_BLOCK",
}
`;

exports[`Ticket actions Ticket actions set ticket availability 1`] = `
Object {
  "payload": Object {
    "available": 99,
    "blockId": "modern-tribe",
  },
  "type": "@@MT/TICKETS/SET_TICKET_AVAILABLE",
}
`;

exports[`Ticket actions Ticket actions set ticket currency positioin 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "currencyPosition": "suffix",
  },
  "type": "@@MT/TICKETS/SET_TICKET_CURRENCY_POSITION",
}
`;

exports[`Ticket actions Ticket actions set ticket currency symbol 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "currencySymbol": "$",
  },
  "type": "@@MT/TICKETS/SET_TICKET_CURRENCY_SYMBOL",
}
`;

exports[`Ticket actions Ticket actions set ticket has been created 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "hasBeenCreated": true,
  },
  "type": "@@MT/TICKETS/SET_TICKET_HAS_BEEN_CREATED",
}
`;

exports[`Ticket actions Ticket actions set ticket has been created 2`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "hasBeenCreated": false,
  },
  "type": "@@MT/TICKETS/SET_TICKET_HAS_BEEN_CREATED",
}
`;

exports[`Ticket actions Ticket actions set ticket has changes 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "hasChanges": true,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_HAS_CHANGES",
}
`;

exports[`Ticket actions Ticket actions set ticket has changes 2`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "hasChanges": false,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_HAS_CHANGES",
}
`;

exports[`Ticket actions Ticket actions set ticket id 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "ticketId": 99,
  },
  "type": "@@MT/TICKETS/SET_TICKET_ID",
}
`;

exports[`Ticket actions Ticket actions set ticket is loading 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "isLoading": true,
  },
  "type": "@@MT/TICKETS/SET_TICKET_IS_LOADING",
}
`;

exports[`Ticket actions Ticket actions set ticket is loading 2`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "isLoading": false,
  },
  "type": "@@MT/TICKETS/SET_TICKET_IS_LOADING",
}
`;

exports[`Ticket actions Ticket actions set ticket is selected 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "isSelected": true,
  },
  "type": "@@MT/TICKETS/SET_TICKET_IS_SELECTED",
}
`;

exports[`Ticket actions Ticket actions set ticket is selected 2`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "isSelected": false,
  },
  "type": "@@MT/TICKETS/SET_TICKET_IS_SELECTED",
}
`;

exports[`Ticket actions Ticket actions set ticket provider 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "provider": "provider",
  },
  "type": "@@MT/TICKETS/SET_TICKET_PROVIDER",
}
`;

exports[`Ticket actions Ticket actions set ticket sold 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "sold": 23,
  },
  "type": "@@MT/TICKETS/SET_TICKET_SOLD",
}
`;

exports[`Ticket actions Ticket details actions set ticket capacity 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "capacity": "10",
  },
  "type": "@@MT/TICKETS/SET_TICKET_CAPACITY",
}
`;

exports[`Ticket actions Ticket details actions set ticket capacity type 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "capacityType": "unlimited",
  },
  "type": "@@MT/TICKETS/SET_TICKET_CAPACITY_TYPE",
}
`;

exports[`Ticket actions Ticket details actions set ticket description 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "description": "The Next Generation of Digital Agency",
  },
  "type": "@@MT/TICKETS/SET_TICKET_DESCRIPTION",
}
`;

exports[`Ticket actions Ticket details actions set ticket end date 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endDate": "2018-01-10",
  },
  "type": "@@MT/TICKETS/SET_TICKET_END_DATE",
}
`;

exports[`Ticket actions Ticket details actions set ticket end date input 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endDateInput": "January 10, 2018",
  },
  "type": "@@MT/TICKETS/SET_TICKET_END_DATE_INPUT",
}
`;

exports[`Ticket actions Ticket details actions set ticket end moment 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endDateMoment": Object {
      "type": "moment",
    },
  },
  "type": "@@MT/TICKETS/SET_TICKET_END_DATE_MOMENT",
}
`;

exports[`Ticket actions Ticket details actions set ticket end moment 2`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endDateMoment": null,
  },
  "type": "@@MT/TICKETS/SET_TICKET_END_DATE_MOMENT",
}
`;

exports[`Ticket actions Ticket details actions set ticket end time 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endTime": "12:34",
  },
  "type": "@@MT/TICKETS/SET_TICKET_END_TIME",
}
`;

exports[`Ticket actions Ticket details actions set ticket end time input 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endTimeInput": "12:34",
  },
  "type": "@@MT/TICKETS/SET_TICKET_END_TIME_INPUT",
}
`;

exports[`Ticket actions Ticket details actions set ticket price 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "price": 99,
  },
  "type": "@@MT/TICKETS/SET_TICKET_PRICE",
}
`;

exports[`Ticket actions Ticket details actions set ticket sku 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "sku": "my-sku",
  },
  "type": "@@MT/TICKETS/SET_TICKET_SKU",
}
`;

exports[`Ticket actions Ticket details actions set ticket start date 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startDate": "2018-01-01",
  },
  "type": "@@MT/TICKETS/SET_TICKET_START_DATE",
}
`;

exports[`Ticket actions Ticket details actions set ticket start date input 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startDateInput": "January 1, 2018",
  },
  "type": "@@MT/TICKETS/SET_TICKET_START_DATE_INPUT",
}
`;

exports[`Ticket actions Ticket details actions set ticket start moment 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startDateMoment": Object {
      "type": "moment",
    },
  },
  "type": "@@MT/TICKETS/SET_TICKET_START_DATE_MOMENT",
}
`;

exports[`Ticket actions Ticket details actions set ticket start moment 2`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startDateMoment": null,
  },
  "type": "@@MT/TICKETS/SET_TICKET_START_DATE_MOMENT",
}
`;

exports[`Ticket actions Ticket details actions set ticket start time 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startTime": "10:00",
  },
  "type": "@@MT/TICKETS/SET_TICKET_START_TIME",
}
`;

exports[`Ticket actions Ticket details actions set ticket start time input 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startTimeInput": "10:00",
  },
  "type": "@@MT/TICKETS/SET_TICKET_START_TIME_INPUT",
}
`;

exports[`Ticket actions Ticket saga actions create new ticket 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
  },
  "type": "@@MT/TICKETS/CREATE_NEW_TICKET",
}
`;

exports[`Ticket actions Ticket saga actions delete ticket 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
  },
  "type": "@@MT/TICKETS/DELETE_TICKET",
}
`;

exports[`Ticket actions Ticket saga actions fetch ticket 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "ticketId": 99,
  },
  "type": "@@MT/TICKETS/FETCH_TICKET",
}
`;

exports[`Ticket actions Ticket saga actions handle ticket start date 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "date": Object {},
    "dayPickerInput": Object {},
  },
  "type": "@@MT/TICKETS/HANDLE_TICKET_START_DATE",
}
`;

exports[`Ticket actions Ticket saga actions handle ticket start date 2`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "date": Object {},
    "dayPickerInput": Object {},
  },
  "type": "@@MT/TICKETS/HANDLE_TICKET_END_DATE",
}
`;

exports[`Ticket actions Ticket saga actions handle ticket start date 3`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "seconds": 1000,
  },
  "type": "@@MT/TICKETS/HANDLE_TICKET_START_TIME",
}
`;

exports[`Ticket actions Ticket saga actions handle ticket start date 4`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "seconds": 1000,
  },
  "type": "@@MT/TICKETS/HANDLE_TICKET_END_TIME",
}
`;

exports[`Ticket actions Ticket saga actions set ticket details 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "details": Object {},
  },
  "type": "@@MT/TICKETS/SET_TICKET_DETAILS",
}
`;

exports[`Ticket actions Ticket saga actions set ticket initial state 1`] = `
Object {
  "payload": Object {
    "attributes": Object {
      "ticketId": 99,
    },
    "clientId": "modern-tribe",
  },
  "type": "@@MT/TICKETS/SET_TICKET_INITIAL_STATE",
}
`;

exports[`Ticket actions Ticket saga actions set ticket temp details 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "tempDetails": Object {},
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_DETAILS",
}
`;

exports[`Ticket actions Ticket saga actions update ticket 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
  },
  "type": "@@MT/TICKETS/UPDATE_TICKET",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp capacity 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "capacity": "10",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_CAPACITY",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp capacity type 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "capacityType": "unlimited",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_CAPACITY_TYPE",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp description 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "description": "The Next Generation of Digital Agency",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_DESCRIPTION",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp end date 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endDate": "2018-01-10",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_END_DATE",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp end date input 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endDateInput": "January 10, 2018",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_END_DATE_INPUT",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp end moment 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endDateMoment": Object {
      "type": "moment",
    },
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_END_DATE_MOMENT",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp end moment 2`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endDateMoment": null,
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_END_DATE_MOMENT",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp end time 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endTime": "12:34",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_END_TIME",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp end time input 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "endTimeInput": "12:34",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_END_TIME_INPUT",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp price 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "price": 99,
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_PRICE",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp sku 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "sku": "my-sku",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_SKU",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp start date 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startDate": "2018-01-01",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_START_DATE",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp start date input 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startDateInput": "January 1, 2018",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_START_DATE_INPUT",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp start moment 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startDateMoment": Object {
      "type": "moment",
    },
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_START_DATE_MOMENT",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp start moment 2`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startDateMoment": null,
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_START_DATE_MOMENT",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp start time 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startTime": "10:00",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_START_TIME",
}
`;

exports[`Ticket actions Ticket temp details actions set ticket temp start time input 1`] = `
Object {
  "payload": Object {
    "blockId": "modern-tribe",
    "startTimeInput": "10:00",
  },
  "type": "@@MT/TICKETS/SET_TICKET_TEMP_START_TIME_INPUT",
}
`;

exports[`Ticket actions Tickets actions close settings 1`] = `
Object {
  "payload": Object {
    "isSettingsOpen": false,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_IS_SETTINGS_OPEN",
}
`;

exports[`Ticket actions Tickets actions open settings 1`] = `
Object {
  "payload": Object {
    "isSettingsOpen": true,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_IS_SETTINGS_OPEN",
}
`;

exports[`Ticket actions Tickets actions set tickets header image 1`] = `
Object {
  "payload": Object {
    "image": 10,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_HEADER_IMAGE",
}
`;

exports[`Ticket actions Tickets actions set tickets initial state 1`] = `
Object {
  "payload": Object {
    "attributes": Object {
      "header": 99,
      "sharedCapacity": 19,
    },
    "clientId": "modern-tribe",
  },
  "type": "@@MT/TICKETS/SET_TICKETS_INITIAL_STATE",
}
`;

exports[`Ticket actions Tickets actions set tickets is settings loading 1`] = `
Object {
  "payload": Object {
    "isSettingsLoading": true,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_IS_SETTINGS_LOADING",
}
`;

exports[`Ticket actions Tickets actions set tickets is settings loading 2`] = `
Object {
  "payload": Object {
    "isSettingsLoading": false,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_IS_SETTINGS_LOADING",
}
`;

exports[`Ticket actions Tickets actions set tickets is settings open 1`] = `
Object {
  "payload": Object {
    "isSettingsOpen": true,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_IS_SETTINGS_OPEN",
}
`;

exports[`Ticket actions Tickets actions set tickets is settings open 2`] = `
Object {
  "payload": Object {
    "isSettingsOpen": false,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_IS_SETTINGS_OPEN",
}
`;

exports[`Ticket actions Tickets actions set tickets provider 1`] = `
Object {
  "payload": Object {
    "provider": "Tribe__Tickets__Commerce__PayPal__Main",
  },
  "type": "@@MT/TICKETS/SET_TICKETS_PROVIDER",
}
`;

exports[`Ticket actions Tickets actions set tickets shared capacity 1`] = `
Object {
  "payload": Object {
    "sharedCapacity": 99,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_SHARED_CAPACITY",
}
`;

exports[`Ticket actions Tickets actions set tickets temp shared capacity 1`] = `
Object {
  "payload": Object {
    "tempSharedCapacity": 99,
  },
  "type": "@@MT/TICKETS/SET_TICKETS_TEMP_SHARED_CAPACITY",
}
`;
